realm/chess.gno

```diff
@@ -450,7 +451,7 @@ func ClaimTimeout(gameID string) string {
 			g.Concluder = &g.White
 		}
 		g.Winner = WinnerNone
-		return g.json()
+		return g.json(), nil
```

Comment:
@thehowl 
What implications does this have for the return type parsing?

Reply:
the function is now unexported.

but anyway, it should just join the return values (with type), with a comma.

Reply:
@zivkovicmilos None, Antonio created a clone without an error which is exported and should work as normal